crates/ide_assists/src/handlers/convert_tuple_struct_to_named_struct.rs

```diff
@@ -0,0 +1,438 @@
+use hir::{Adt, ModuleDef, Struct};
```

Comment:
Prefer prefixing these with `hir::` where they are used instead of importing, https://github.com/rust-analyzer/rust-analyzer/blob/master/docs/dev/style.md#import-style